context/app/static/js/components/Detail/provenance/ProvGraph/ProvGraph.spec.js

```diff
@@ -0,0 +1,99 @@
+import React from 'react';
+import { fireEvent, waitFor } from '@testing-library/react';
+import { rest } from 'msw';
+import { setupServer } from 'msw/node';
+
+// eslint-disable-next-line import/no-unresolved
+import { render, screen, appProviderEndpoints } from 'test-utils/functions';
+import sampleProv, { sampleDescendantsProv } from './fixtures/sample_prov';
+import ProvGraph from './ProvGraph';
+
+const server = setupServer(
+  rest.post(`/${appProviderEndpoints.elasticsearchEndpoint}`, (req, res, ctx) => {
+    const descendantData = {
+      hits: {
+        hits: [
+          {
+            _source: {
+              immediate_descendants: [
+                {
+                  uuid: sampleDescendantsProv[0].uuid,
+                },
+                {
+                  uuid: sampleDescendantsProv[1].uuid,
+                },
+              ],
+            },
+          },
+        ],
+      },
+    };
+    return res(ctx.json(descendantData), ctx.status(200));
+  }),
+  rest.get(
+    `/${appProviderEndpoints.entityEndpoint}/entities/${sampleDescendantsProv[0].uuid}/provenance`,
+    (req, res, ctx) => {
+      return res(ctx.json(sampleDescendantsProv[0].prov), ctx.status(200));
+    },
+  ),
+  rest.get(
+    `/${appProviderEndpoints.entityEndpoint}/entities/${sampleDescendantsProv[1].uuid}/provenance`,
+    (req, res, ctx) => {
+      return res(ctx.json(sampleDescendantsProv[1].prov), ctx.status(200));
+    },
+  ),
+);
+
+beforeAll(() => server.listen({ onUnhandledRequest: 'error' }));
+afterEach(() => server.resetHandlers());
+afterAll(() => server.close());
+
+test('should display the correct initial nodes', () => {
+  const sampleEntityText = 'Sample - HBM666.CHPF.373';
+  const nodesText = [
+    'hubmap:entities/73bb26e4-ed43-11e8-8f19-0a7c1eab007a',
+    'Register Donor Activity - HBM398.NBBW.527',
+    'Donor - HBM547.NCQL.874',
+    'Create Sample Activity - HBM665.NTZB.997',
+    sampleEntityText,
+  ];
+
+  render(<ProvGraph provData={sampleProv} />);
+
+  nodesText.forEach((text) => expect(screen.getByText(text)).toBeInTheDocument());
+});
+
+test('should selected node information in detail pane and show immediate descendants when show derived entities button is clicked', async () => {
```

Comment:
```suggestion
test('should select node information in detail pane and show immediate descendants when showing derived entities button is clicked', async () => {
```
selected -> select and show -> showing?

Reply:
'should display selected node information in detail pain and show immediate descendants when "Show Derived Entities" button is clicked', perhaps?

Reply:
sounds good!